common/common.py

```diff
@@ -11,16 +11,14 @@ def get_group(athena_group: AthenaGroup, key: str = None) -> Group:
     group_keys = list(athena_group.keys())
     if key is None:
         key = group_keys[0]
-    else:
-        key = key.replace("-", "_")
 
     try:
         return extract_athenagroup(athena_group.groups[key])
     except KeyError as e:
         raise KeyError(f"{key} not in {group_keys}") from e
 
 
-def read_all_groups(dat_file: str, key: str = None) -> "dict[str, Group]":
+def read_all_groups(dat_file: str) -> "dict[str, Group]":
```

Comment:
have not seen this function referenced anywhere.

Reply:
It's used on line 99 in `larch_athena.py`, which isn't changed as part of this PR. It doesn't pass `key`, and `key` is overwritten in the for loop so it was serving no purpose. Probably got copy pasted when first writing this function.